2022-06-09 16:26:25,430 INFO    MainThread:15912 [wandb_setup.py:_flush():76] Loading settings from /Users/zhangqiming/.config/wandb/settings
2022-06-09 16:26:25,430 INFO    MainThread:15912 [wandb_setup.py:_flush():76] Loading settings from /Users/zhangqiming/Desktop/RL_research/Reinforcement-Learning/project/wandb/settings
2022-06-09 16:26:25,430 INFO    MainThread:15912 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2022-06-09 16:26:25,431 INFO    MainThread:15912 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'project/exmaple.py', 'program': '/Users/zhangqiming/Desktop/RL_research/Reinforcement-Learning/project/exmaple.py'}
2022-06-09 16:26:25,431 INFO    MainThread:15912 [wandb_init.py:_log_setup():428] Logging user logs to /Users/zhangqiming/Desktop/RL_research/Reinforcement-Learning/project/wandb/run-20220609_162625-35vpgs2m/logs/debug.log
2022-06-09 16:26:25,431 INFO    MainThread:15912 [wandb_init.py:_log_setup():429] Logging internal logs to /Users/zhangqiming/Desktop/RL_research/Reinforcement-Learning/project/wandb/run-20220609_162625-35vpgs2m/logs/debug-internal.log
2022-06-09 16:26:25,432 INFO    MainThread:15912 [wandb_init.py:init():462] calling init triggers
2022-06-09 16:26:25,432 INFO    MainThread:15912 [wandb_init.py:init():466] wandb.init called with sweep_config: {}
config: {}
2022-06-09 16:26:25,432 INFO    MainThread:15912 [wandb_init.py:init():515] starting backend
2022-06-09 16:26:25,432 INFO    MainThread:15912 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-06-09 16:26:25,450 INFO    MainThread:15912 [backend.py:ensure_launched():217] starting backend process...
2022-06-09 16:26:25,879 INFO    MainThread:15912 [backend.py:ensure_launched():222] started backend process with pid: 15922
2022-06-09 16:26:25,887 INFO    MainThread:15912 [wandb_init.py:init():525] backend started and connected
2022-06-09 16:26:25,936 INFO    MainThread:15912 [wandb_init.py:init():596] updated telemetry
2022-06-09 16:26:25,983 INFO    MainThread:15912 [wandb_init.py:init():628] communicating run to backend with 30 second timeout
2022-06-09 16:26:55,992 INFO    MainThread:15912 [interface_shared.py:_communicate_run():326] couldn't get run from backend
2022-06-09 16:26:55,993 ERROR   MainThread:15912 [wandb_init.py:init():636] backend process timed out
2022-06-09 16:26:55,994 ERROR   MainThread:15912 [wandb_init.py:init():647] encountered error: Error communicating with wandb process
try: wandb.init(settings=wandb.Settings(start_method='fork'))
or:  wandb.init(settings=wandb.Settings(start_method='thread'))
For more info see: https://docs.wandb.ai/library/init#init-start-error
2022-06-09 16:27:21,983 INFO    MainThread:15912 [wandb_init.py:teardown():276] tearing down wandb.init
